src/main/java/programminglife/ProgrammingLife.java

```diff
@@ -26,18 +30,26 @@
     private static final String TB_DATA = DATA_FOLDER + "real/TB10.gfa";
     private static final String HUMAN_DATA = DATA_FOLDER + "real/chr19.hg38.w115.gfa";
```

Comment:
Consider removing these, as paths may differ per project. These paths should be a run configuration in your IDE.